backend/src/packages/chats/chat.controller.ts

```diff
@@ -93,6 +119,32 @@ class ChatController extends BaseController {
         );
       },
     });
+
+    this.addRoute({
+      path: ChatsApiPath.$CHAT_ID_MESSAGES,
+      method: 'POST',
+      handler: (options) => {
+        return this.createMessage(
+          options as APIHandlerOptions<{
+            body: ChatMessageCreateRequestDto;
+            params: ChatMessagesUrlParameter;
```

Comment:
let's remove this type and define params directly, this params are too specific
```suggestion
            params: { id: string };
```